Sming/Makefile

```diff
@@ -177,6 +177,12 @@ ENABLE_ESPCONN ?= 0
 ifeq ($(ENABLE_CUSTOM_LWIP), 1)
 	THIRD_PARTY_DATA += third-party/esp-open-lwip/Makefile.open
 	EXTRA_INCDIR += third-party/esp-open-lwip/include
+	EXTRA_CFLAGS_LWIP = -I../../system/include -I../../Wiring
+	ENABLE_LWIPDEBUG ?= 1
```

Comment:
Sure I can. Right now it prints only errors that are related to tcp reset which I find very helpful and are not shoiwing up very often but of course I can disable it maybe better for more people this way